sophus/so3.hpp

```diff
@@ -585,7 +585,7 @@ class SO3 : public SO3Base<SO3<Scalar_, Options>> {
     using std::sin;
     using std::sqrt;
     Scalar theta_sq = omega.squaredNorm();
-    *theta = sqrt(theta_sq);
```

Comment:
This I believe we should leave as is, and hence the derivative will of sqrt(x) at x=0 will be inf - following @NikolausDemmel's argument.

Reply:
n is also used on line 281